aspnetcore/tutorials/your-first-mac-aspnet.md

```diff
@@ -1,81 +1,71 @@
+## Scaffolding applications using dotnet new
 
-Install the necessary yeoman generators and bower using npm.
+We will be using `dotnet new` to generate a new web application using the "Empty Web Template". Create a working directory for your project called _firstmacapp_ and change the directory to the newly created folder.
 
-```console
-npm install -g yo generator-aspnet bower
-   ```
+Start Visual Studio Code and open the _firstmacapp_ folder. Press Ctrl + '`' (the back-quote character) to open an embedded terminal in VS Code. (Alternatively, you can use a separate terminal window, if you prefer).
 
-Run the ASP.NET Core generator
+Run the `dotnet new` command to create a new web application, passing the `mvc` parameter as the template type.
 
 ```console
-yo aspnet
-   ```
-
-* Select **Web Application Basic [without Membership and Authorization]** and tap Enter
-
-* Select Bootstrap (3.3.6) as the UI framework and tap Enter
+dotnet new mvc
+```
 
-* Use "MyFirstApp" for the app name and tap Enter
-
-When the generator completes scaffolding the files, it will instruct you to restore, build, and run the application.
+When the CLI command completes; the following output and files are produced.
 
 ```console
-Your project is now created, you can use the following commands to get going
-       cd "MyFirstApp"
-       dotnet restore
-       dotnet build (optional, build will also happen with it's run)
-       dotnet run
-   ```
-
-## Developing ASP.NET Core Applications on a Mac With Visual Studio Code
+Content generation time: 79.6691 ms
+The template "Empty ASP.NET Core Web Application" created successfully.
+```
 
-* Start **Visual Studio Code**
+* Startup.cs : [Startup Class](../fundamentals/startup.md) - class configures the request pipeline that handles all requests made to the application.
+* Program.cs : [Program Class](../fundamentals/) that contains the Main entry point of the application.
+* firstmacapp.csproj : [Project file](https://docs.microsoft.com/en-us/dotnet/articles/core/preview3/tools/csproj) MSBuild Project file format for ASP.NET Core applications. Contains Project to Project references, NuGet References and other project related items.
+* appsettings.json / appsettings.Development.json : Environment base app settings configuration file. [See Configuration](../fundamentals/configuration).
+* bower.json : Bower package dependencies for the project.
+* .bowerrc : Bower configuration file which defines where to install the components when Bower downloads the assets.
+* bundleconfig.json : configuration files for bundling and minifying front-end JavaScript and CSS assets.
+* Views : Contains the Razor views for the Controllers as well as Shared Views. Views are the components that display the app's user interface (UI). Generally, this UI displays the model data.
```

Comment:
**as well** banned
 Contains the Razor views for the controllers ~~as well as~~ and  shared views.
Why can't it be
 Contains the [Razor views](xref:mvc/views/razor)